src/Shared/UnitTests/Vacancies.Client/Application/VacancyValidation/SingleField/TrainingProviderUkprnValidationTests.cs

```diff
@@ -115,5 +117,60 @@ public void TrainingProviderUkprnMustBe8Digits(long? ukprn)
             result.Errors[0].ErrorCode.Should().Be("99");
             result.Errors[0].RuleId.Should().Be((long)VacancyRuleSet.TrainingProvider);
         }
+
+        [Fact]
+        public void ErrorIfProviderVacancyDoesNotHaveEmployerPermission()
+        {
+            const long ukprn = 12345678;
+            const string employerAccountId = "employer-account-id";
+            const long legalEntityId = 1234;
+
+            var vacancy = new Vacancy
+            {
+                OwnerType = OwnerType.Provider,
+                TrainingProvider = new TrainingProvider { Ukprn = ukprn },
+                EmployerAccountId = employerAccountId,
+                LegalEntityId = legalEntityId
+            };
+
+            MockTrainingProviderSummaryProvider.Setup(p => p.GetAsync(12345678)).ReturnsAsync(new TrainingProviderSummary());
```

Comment:
Can you use the ukprn const in this call setup please like you have in the test method below.

Reply:
👍